retrieval/reports/archie_toolcall_probe.py

```
@@ -0,0 +1,61 @@
+           "Use it by writing: CALL web_search(\"your query\")")
+
+ASK = "What is the standard tapping drill diameter for an M12 x 1.75 threaded hole?"
+
+CASES = [("json-schema", SYS_JSON), ("xml-tag", SYS_XML), ("minimal", SYS_MIN)]
+
+def parseable(text):
+    """Did it emit something a dispatcher could actually execute?"""
+    hits = []
+    try:
+        m = re.search(r'\{[^{}]*"tool"\s*:\s*"web_search"[^{}]*\{[^{}]*\}[^{}]*\}', text, re.S)
+        if m: json.loads(m.group(0)); hits.append("json")
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)